contracts/rust/Cargo.toml

```diff
@@ -8,27 +8,33 @@ edition = "2021"
 [dependencies]
 anyhow = "1.0.71"
 ark-bn254 = "0.4.0"
-ark-ec = { version = "0.4.2" }
+ark-ec = "0.4.2"
 ark-ff = "0.4.2"
+ark-poly = "0.4.2"
 ark-serialize = "0.4.2"
 ark-std = { version = "0.4.0", default-features = false }
 async-compatibility-layer = { git = "https://github.com/EspressoSystems/async-compatibility-layer", tag = "1.4.1", features = [
     "logging-utils",
 ] }
 async-std = "1.12.0"
+clap = { version = "^4.4", features = ["derive"] }
 contract-bindings = { path = "../../contract-bindings" }
+crs = { git = "https://github.com/alxiong/crs" }
 digest = { version = "0.10", default-features = false, features = ["alloc"] }
-ethers = { version = "2.0.4", features = ["abigen"] }
+ethers = { version = "2.0.4" }
 ethers-providers = "2.0.4"
 ethers-solc = "2.0.4"
 hex = "0.4.3"
-
-# This crate requires a newer version than 0.1.2-patch.1 that's used elsewhere
-# in this workspace. However not pinning any revision currently breaks
-# dependency resolution.
-jf-primitives = { git = "https://github.com/EspressoSystems/jellyfish", features = [
-    "std",
-] }
-jf-utils = { git = "https://github.com/EspressoSystems/jellyfish", rev = "483fb97324d42724720f9155f200973a3b83a7b4" }
+itertools = "0.10.3"
+jf-plonk = { git = "https://github.com/EspressoSystems/jellyfish", features = ["test-apis", "test-srs"], branch = "more-test-apis" }
```

Comment:
Should we stay pinned to this branch?

Reply:
I plan to pin to a rev on the main branch once [this PR](https://github.com/EspressoSystems/jellyfish/pull/398) is merged.

Reply:
done in [af7266c](https://github.com/EspressoSystems/espresso-sequencer/pull/722/commits/af7266c6b8bca80d4be59103cbe40187092c798f)